toolchain/driver/clang_runner.cpp

```diff
@@ -146,13 +146,15 @@ auto ClangRunner::Run(llvm::ArrayRef<llvm::StringRef> args) -> bool {
   // Note the subprocessing will effectively call `clang -cc1`, which turns into
   // `carbon-busybox clang -cc1`, which results in an equivalent `clang_main`
   // call.
-  driver.CC1Main = [](llvm::SmallVectorImpl<const char*>& cc1_args) -> int {
+  std::function<int(llvm::SmallVectorImpl<const char*> & cc1_args)> cc1_main =
```

Comment:
```suggestion
  auto cc1_main =
```

Does this need to explicitly be a std::function, or is it enough to store the lambda directly?

Reply:
Good point, changing to auto.